src/components/Watch.svelte

```diff
@@ -86,6 +92,14 @@
     });
   };
   $: setTimeout(() => changeSide($videoSide, $chatSplit), 0);
+
+  function orientationChangeCallback() {
+    $videoSide = ($autoVertical && window.innerHeight > window.innerWidth) ? VideoSide.TOP : $videoSideSetting;
```

Comment:
This logic to change ``videoSide`` which is distributed across multiple files is getting really confusing. You should refactor the logic of ``videoSide`` and turn it into a [readable store](https://svelte.dev/tutorial/readable-stores) so all the mutations are managed at one place and we know what's happening.